python-sdk/tutorials/automl-with-azureml/generate_workflows.py

```diff
@@ -32,6 +35,17 @@ def main():
 def write_notebook_workflow(notebook, notebook_folder, cron_schedule):
     notebook_name = notebook.replace(".ipynb", "")
     creds = "${{secrets.AZ_AE_CREDS}}"
+
+    run_update_env = ""
+    update_yml_file = (
+        f"python-sdk/tutorials/automl-with-azureml/{notebook_folder}/{UPDATE_ENV_YML}"
+    )
+    if os.path.exists(os.path.join(notebook_folder, UPDATE_ENV_YML)):
+        run_update_env = f"""
+    - name: update conda env with the update_env.yml
```

Comment:
maybe put a comment here saying that we update the environment with notebook specific dependencies.

Reply:
added